keywording/text.xml

```diff
@@ -196,8 +196,8 @@ a dependency of a package already keyworded.
 
 <p>
 Do <b>not</b> commit straight to <c>arch</c> ("stable"). If it is necessary
-to do so, when using <c>repoman</c>, use
-<c>repoman commit --straight-to-stable</c> and <b>not</b> <c>--force</c> as
+to do so, when using <c>pkgdev</c>, use
+<c>pkgdev commit --ask --scan</c> and <b>not</b> by dropping <c>--scan</c> as
```

Comment:
You can `pkgdev commit` straight to stable just fine, it won't nag until you run `--scan`. Just saying for normal workflow you recommend `pkgdev commit` and this shouldn't be different, ie, you often catch the error only when doing `--scan`.

Reply:
Yeah. So how should I change this paragraph? repoman would have blocked committing straight to stable; pkgdev doesn't (unless you use --scan) and instead assumes that you'll run `pkgcheck scan` later.

I think I should probably just remove the paragraph.

Reply:
Just keep the first sentence.

Reply:
WFM.